js/orderForm/orderForm.js

```diff
@@ -0,0 +1,59 @@
+import { $ } from '../base/base.js';
+import { PRODUCT_STORAGE_KEY } from '../base/constant.js';
+import validateForm from '../component/validate.js'
+import toast from '../component/toast.js';
+
+const form = $('#formOrder');
+
+
+const formOrder = {
+    handleEvents: function () {
+        form.addEventListener("submit", event => {
+            const products =  JSON.parse(localStorage.getItem(PRODUCT_STORAGE_KEY)) || []
+            const productIds = products.map((product) => {
+                return product.id
+            })
+            let formData = {
+                name: form.name.value,
+                email: form.email.value,
+                address: form.address.value,
+                phone: form.phoneNumber.value,
+                productIds: productIds
+            }
+            event.preventDefault();
+            if(validateForm(form)) {
+                if (productIds.length > 0) {
```

Comment:
if else lồng nhau nhiều quá
```suggestion
            if(validateForm(form) && productIds.length > 0) {
```
Như này cũng được mà. K cần phần else return false ở sau luôn